image_tools/include/image_tools/showimage.hpp

```diff
@@ -0,0 +1,75 @@
+// Copyright 2015 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef IMAGE_TOOLS__SHOW_CONFIG_COMPONENT_HPP_
+#define IMAGE_TOOLS__SHOW_CONFIG_COMPONENT_HPP_
+
+#include <cstdio>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "opencv2/highgui/highgui.hpp"
```

Comment:
It doesn't appear that theses OpenCV headers are needed here. Best move them to the `cpp` file where they are needed. Same goes for any other headers that are only needed in the `cpp` file.